.doc/requirements.txt

```diff
@@ -3,6 +3,4 @@ ipython
 ipykernel
 rtds_action
 sphinx_rtd_theme
-
-
-
+recommonmark
```

Comment:
i had to manually add this... not sure why CI dosent need it

Reply:
I don't know why it doesn't cause a problem either